packages/std-client/package.json

```diff
@@ -8,16 +8,21 @@
     "directory": "packages/std-client"
   },
   "license": "MIT",
+  "type": "module",
```

Comment:
I changed this so I could move `codegen.mts` to `codegen.ts` and align this with the rest of our packages. I ran `templates/minimal` with this and it seemed to work just fine.